addons/binding/org.openhab.binding.tellstick/src/main/java/org/openhab/binding/tellstick/internal/live/xml/DataTypeValue.java

```diff
@@ -47,6 +50,24 @@ public void setValue(String data) {
 
     @Override
     public String toString() {
```

Comment:
Normally I would expect such as methods as `toString` as the final method of a class.